ee/modules/160-multitenancy-manager/crds/doc-ru-projecttemplate.yaml

```
@@ -0,0 +1,68 @@
+    - name: v1alpha2
+      schema:
+        openAPIV3Schema:
+          description: |
+            Описывает шаблон окружения.
+
+            Этот на этот ресурс ссылается ресурс [Project](cr.html#project).
+          properties:
+            spec:
+              description: ""
+              properties:
+                namespaceMetadata:
+                  description:
+                    Лейблы и аннотации, которые применяются к создаваемым
+                    пространствам имен при настройке окружения.
+                  properties:
+                    annotations:
+                      description: ""
+                    labels:
+                      description: ""
+                valuesSchema:
+                  description:
+                    Спецификация OpenAPI для значений (`values`) шаблона
+                    (параметр [template](#projecttemplate-v1alpha2-spec-template)).
+                template:
+                  description: |
+                    Шаблоны ресурсов в формате `helm`, которые требуется создать при настройке нового окружения.
+
+                    Шаблоны совместимы со всеми функциями `helm`. Читайте подробнее про [создание изолированных окружений](usage.html#создание-изолированного-окружения)...
+
+                    В шаблонах доступны также следующие параметры (`values`):
+                    - `{{ .projectName }}` — имя `Project` (окружения), для которого рендерится шаблон;
+                    - `{{ .templateName }}` — имя `ProjectTemplate`;
+                    - `{{ .templateValues }}` — словарь пользовательских значений, описанных в параметре [.spec.valuesSchema](cr.html#projecttemplate-v1alpha2-spec-valuesSchema) и определенных в параметре [.spec.template](cr.html#project-v1alpha2-spec-template).
+
+                    > **Внимание!** Указывать поля `.metadata.namespace` для объектов необязательно,
+                    > так как они автоматически проставятся из названия окружения (custom resource `Project`).
+                subjects:
+                  description:
+                    Список ServiceAccount'ов, групп (`Group`) и пользователей
+                    (`User`) для предоставления доступа к созданному окружению.
+                  items:
+                    description: ""
+                    properties:
+                      kind:
+                        description:
+                          Kind целевого ресурса для применения доступа к
+                          окружению (`ServiceAccount`, `Group` или `User`).
+                      name:
+                        description: Имя целевого ресурса для применения доступа к окружению.
+                      namespace:
+                        description: |
+                          Пространство имен (`namespace`) целевого ресурса для применения доступа к окружению.
+
+                          Требуется только при использовании ServiceAccount'а из другого пространства имен (`namespace`).
+                      role:
```

```suggestion
                      accessLevel:
```

It's definitely not a role